big_tests/tests/websockets_SUITE.erl

```diff
@@ -161,3 +167,10 @@ escape_attrs(Config) ->
         special_chars_helper:check_attr_from_to(Geralt, Carol)
 
     end).
+
+instrumentation_events() ->
+    % because mod_websockets is mongoose_http_handler, it starts http_handler_instrumentation globally.
+    % To avoid conflicts with cases when the module is both mongoose_http_handler and gen_mod,
+    % we use http_handler_instrumentation/0 name instead of instrumentation/0.
+    Specs = rpc(mim(), mod_websockets, http_handler_instrumentation, []),
```

Comment:
We could extend `instrument_helper:declared_events` to work also for the global callback `instrumentation/0`.